modelci/env-backend.env

```diff
@@ -0,0 +1,3 @@
+BACKEND_CORS_ORIGINS="http://localhost,http://localhost:3000,http://localhost:8080,https://localhost:3000,https://localhost:8080"
+PROJECT_NAME=modelci
+SECRET_KEY=2a6c03b9ca06cd8fc3cf506f0ba924cb735f15918d54758426fd7282366a5e19
```

Comment:
should we put here?

Reply:
The secret key used for production. The RESTful server structure is simplified from FastAPI project template https://github.com/tiangolo/full-stack-fastapi-postgresql